tests/antithesis/xsvm/main.go

```diff
@@ -92,6 +96,8 @@ func main() {
 		}
 	}
 
+	lifecycle.SetupComplete(fmt.Sprintf("Initialized %d workers", NumKeys))
```

Comment:
Seems like this supports structure, should this be
```suggestion
	lifecycle.SetupComplete(map[string]any{
	    "msg": "initialized workers"
	    "numWorkers": NumKeys,
	})
```